### ISSUE TITLE
Option to suppress progress bar output of ti_lookup.lookup_ioc()

### ISSUE THREAD

**Issue by pstirparo** [microsoft/msticpy#629]
**Describe the solution you'd like**
if I have a loop with a bunch/many IOCs I want to enrich like 
`ti_lookup.lookup_ioc(ip, ioc_type="ipv4", providers=["GreyNoise"])`
I may not want to see tons of progress bars filling up my notebook.

of course is not a high priority request as this is not blocking anything.
thanks

**Comment by ianhelle** [comment]
Should be fixed in #631. We expect this to go out in 2.3.1

**Comment by ianhelle** [comment]
syntax is
```python
ti_lookup.lookup_ioc(ip, ioc_type="ipv4", providers=["GreyNoise"], progress=False)
```
